web/src/popover_menus.js

```diff
@@ -921,28 +922,30 @@ export function initialize() {
     });
 
     // Starred messages popover
-    register_popover_menu(".starred-messages-sidebar-menu-icon", {
-        ...left_sidebar_tippy_options,
-        onMount(instance) {
-            const $popper = $(instance.popper);
-            popover_instances.starred_messages = instance;
-
-            $popper.one("click", "#unstar_all_messages", () => {
-                starred_messages_ui.confirm_unstar_all_messages();
-                instance.hide();
+    function starred_messages_onMount(instance) {
```

Comment:
This should be probably kept to snake_case--`starred_messages_on_mount`, but there might even be a better name for this function than that.

Reply:
Changed names to `snake_case` for now. Here are some options for a better name:

1) starred_messages_handlers
2) starred_messages_sidebar_menu_options
3) starred_messages_sidebar_menu_options_handler